tools/state-parts-dump-check/src/cli.rs

```diff
@@ -502,10 +506,10 @@ async fn run_single_check(
         .set(num_parts as i64);
 
     if num_parts < total_required_parts {
-        tracing::info!("total state parts required: {} < number of parts already dumped: {}, waiting for all parts to be dumped", total_required_parts, num_parts);
+        tracing::info!("epoch_height: {}, shard_id: {}, total state parts required: {} > number of parts already dumped: {}, waiting for all parts to be dumped", epoch_height, shard_id, total_required_parts, num_parts);
```

Comment:
I don't have a strong opinion, but I prefer
```
tracing::info!(epoch_height, total_required_parts)
```

to
```
tracing::info!("epoch_height: {}, total_required_parts: {}", epoch_height, total_required_parts);
```

or even to
```
tracing::info!("epoch_height: {epoch_height}, total_required_parts: {total_required_parts}");
```

Reply:
addressed.